src/components/IssueSearchAdvanced.vue

```diff
@@ -13,10 +13,34 @@ const props = defineProps<{
   isLoading: boolean;
 }>();
 
-const route = useRoute();
-
 const emit = defineEmits(['filter:save']);
 
+const isNonEmptyDescriptionSelected = ref(false);
+
+const descriptionParamValue = computed(() => facets.value.find(facet => facet.field === 'cve_description')?.value);
```

Comment:
Saving the entire `facet` instead of the value you can reuse it also on line 23
```suggestion
const descriptionParamValue = computed(() => facets.value.find(facet => facet.field === 'cve_description'));
```

Reply:
I thought the same but on the watcher (line 23) I need also to update the value in some cases so I need a writable reference.

Reply:
Well you could use the index